packages/geo-diary/src/components/ui/contact-drawer/aml-check-row.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react'
+import IconButton from '../icon-button'
+import Grid, { Col } from '../grid'
+import * as styles from './__styles__'
+import { Label } from '@reapit/elements/v3'
+
+interface AmlCheckContentProps {
+  contactId: string
+  name: string
```

Comment:
Should this be string or undefined? Istead of empty string?

Reply:
It doesn't really matter in this case - either way I will render it's container even without content so it doesn't break the flow. TBH, it's _very_ unlikely we would see a contact record without a name!